### PR TITLE
Validating course verification deadline

### PR DESCRIPTION
**Pull request by clintonb**
This field is validated to ensure the verification deadline always occurs after the course seats' upgrade deadlines.

XCOM-581

@rlucioni @jimabramson @AlasdairSwan

### CODE REVIEW

File: ecommerce/static/js/models/course_model.js [hunk 1]
```diff
@@ -65,6 +65,22 @@ define([
...
                     msg: gettext('You must select a course type.')
+                },
+                verification_deadline: function (value) {
+                    // No validation is needed for empty values
+                    if (_.isEmpty(value)) {
+                        return;
+                    }
+
+                    // Find seats where the verification deadline occurs before the upgrade deadline.
+                    var invalid = _.some(this.seats(), function (seat) {
```
Comment (AlasdairSwan) [inline ecommerce/static/js/models/course_model.js]: var should be declared at the top of the function.
Also, should this be its own function, i.e. would it be reused elsewhere?

Comment (clintonb) [inline ecommerce/static/js/models/course_model.js]: I cannot think of where else we might perform this validation.

Declaration updated.

**Comment by AlasdairSwan** [comment]
Once tests pass :+1:

**Comment by rlucioni** [comment]
Posthumous :+1: .

**Pull request merged**